# LaneDistance.msg
Header header
bool detected
float32 lateral_distance  # lateral distance in meters
float32 forward_distance  # forward distance in meters